TestApp/Scripts/experience.js

```diff
@@ -48,6 +48,7 @@ function CreateInputHandling(scene) {
 
 var engine = new BABYLON.NativeEngine();
 var scene = new BABYLON.Scene(engine);
+engine.getHostDocument = function () { return {}; }
```

Comment:
Not sure if I missed something. Can you explain what this is for?

Reply:
This is a hack I did to get things going. I've talked to @sebavan to fix it properly.